cmd/broker/ingress/main.go

```diff
@@ -18,28 +18,21 @@ package main
 
 import (
 	"flag"
-	"fmt"
-	"log"
+	"knative.dev/pkg/metrics"
```

Comment:
Run this through `goimports`, the top section is normally only used by standard go library packages.

Reply:
Done, thanks!